src/views/AnnualReport.vue

```diff
@@ -522,6 +521,10 @@ export default class AnnualReport extends Mixins(CommonMixin, DateMixin, FilingM
   saveErrors = []
   saveWarnings = []
 
+  // Enums for Template
+  readonly AuthorizedActions = AuthorizedActions
+  readonly IsAuthorized = IsAuthorized
```

Comment:
Not needed if not used in template.

And you don't need to use "this." in the script, below.